iceoryx_hoofs/include/iceoryx_hoofs/internal/objectpool/objectpool.hpp

```diff
@@ -423,7 +423,8 @@ class ObjectPool
         return m_cellInfo[index].data;
     }
 
-  private:
+  protected:
+    // XXX: Protected for unit tests.
```

Comment:
please remove the `XXX`. Just the comment is sufficient

Reply:
done